addon/mixins/store.js

```diff
@@ -28,16 +28,22 @@ var StoreMixin = Ember.Mixin.create({
    * @method push
    * @inheritDoc
    */
-  push: function (typeName/*, data, _partial*/) {
-    var res = this._super.apply(this, arguments), id, type, adapter;
-    if (this._pushSubscribes && res && (id = res.get('id'))) {
-      type = this.modelFor(typeName);
```

Comment:
You're missing the test on `_pushSubscribes` on the refactored code. I am on my phone and that is a long time I did not work on this addon but I believe it has to be there because depending on settings it should or not subscribe